docs/install.md

```diff
@@ -30,11 +30,43 @@ Coder publishes the following system packages [in GitHub releases](https://githu
 - .rpm (Fedora, CentOS, RHEL, SUSE)
 - .apk (Alpine)
 
-Once installed, you can run Coder as a system service:
+## Configure
```

Comment:
This doc is already damn long. How about we create a new Configuring page under the Install page?

Reply:
I'm good with that. Is there a reason we don't have `install.md` under the `/install` dir?

Reply:
It's an odd pattern that we've adopted. It's the same with ides.md and templates.md.